### DISCUSSION TITLE
Wifi Hotspot turns off when screen is locked.

### DISCUSSION THREAD

**Discussion by 7gxycn08** [Mygod/VPNHotspot#634] [Q&A]
On Samsung tab S8 Ultra hotspot keeps turning off on it's own after a while when i lock the screen.
The app is set to unrestricted and root is granted. Tried setting Wifi timeout is set to lowest latency and always on but it still turns off hotspot on it's own.

Help.

**Comment by Mygod** [comment]
Try this: Go to Wi-Fi hotspot settings in this app (top right corner in the main page) and turn off relevant settings.

**Reply by 7gxycn08** [reply]
The notification says "Inactive: swlan0" Says TETHER_ERROR_UNTETHER_IFACE_ERROR
idle timeout is disabled.
